### ISSUE TITLE
How to beat analytic.press?

### ISSUE THREAD

**Issue by MioVisman** [uBlockOrigin/uAssets#6900]
### URL(s) where the issue occurs

`http://rh.adsh.org.ua/search.php?action=show_recent`
`And many other Russian-language sites`

### Describe the issue

The analytic.press statistics are connected on the sites by replacing the original js scripts through a 307 redirect. See screenshots.
If analytic.press is blocked, the original script will not be loaded :(

### Screenshot(s)

![analyticpress_for_github](https://user-images.githubusercontent.com/5917956/73354023-4a423700-42c7-11ea-841e-a3c685b34dce.png)

### Versions

- Browser/version: Firefox 72.0.2
- uBlock Origin version: 1.24.2

### Settings

- default

### Notes

```
Request URL: http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js
Request method: GET
Status code: 307

HTTP/1.1 307 Temporary Redirect
Pragma: no-cache
Cache-Control: no-store, no-cache, must-revalidate, max-age=0
Connection: keep-alive
Content-Type: text/plain
Location: http://r.analytic.press/?id=85c27bce8183639953238dd557c6854a102f&reg=24&orig=http%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fjquery%2F1.10.2%2Fjquery.min.js
Content-Length: 1
```

**Comment by llacb47** [comment]
Wow, never seen that before.

**Comment by mapx-** [comment]
maybe needing an account or russian ip, I get no `analytic.press`

**Comment by MioVisman** [comment]
I do not know. Now I used a proxy server in Germany, I got the answer:
![analyticpress_for_github2](https://user-images.githubusercontent.com/5917956/73356005-ea01c400-42cb-11ea-9164-74d362c6dd63.png)
![ip](https://user-images.githubusercontent.com/5917956/73356207-65637580-42cc-11ea-9a20-049aa9f47eb7.png)

**Comment by mapx-** [comment]
and `analytic.press` seems to be a parked domain

**Comment by MioVisman** [comment]
Ip address analytic.press sends me to the USA https://ipinfo.io/162.255.119.38
But his subdomains (from which the scripts are loaded) send me to Moscow https://ipinfo.io/195.19.216.34
![1951921634](https://user-images.githubusercontent.com/5917956/73358704-cf324e00-42d1-11ea-9dd5-028421f8e87c.png)

**Comment by llacb47** [comment]
@mapx- See this `http://r.analytic.press/?id=85c27bce8183639953238dd557c6854a102f&reg=24&orig=http%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fjquery%2F1.10.2%2Fjquery.min.js` from original post headers. But if no one can reproduce, maybe it is malware on your machine

**Comment by mapx-** [comment]
yes, I see the script but still cannot reproducing ..
@MioVisman could you try:
`rh.adsh.org.ua##+js(set, encodeURIComponent, trueFunc)`

**Comment by MioVisman** [comment]
Now in Google, when searching for data about the ip address, I came across the topic: **Rostelecom replaces js-files?** https://searchengines.guru/showthread.php?t=1001505
A similar situation is described there.
In 2018, 307 redirects to r.rtkrtb.com and other subdomains.
In 2019, 307 redirects to p.analytic.host and r.analytic.host.
And at the end of 2019, redirects went to p.analytic.press and r.analytic.press.

So this is apparently a local problem for 1 provider. Rostelecom - the largest Russian provider of digital services and services :(

**Comment by llacb47** [comment]
Oh so it is the ISP doing it...

**Comment by uBlock-user** [comment]
Accessing on `http://`, your ISP is intercepting the domain and causing redirects.

**Comment by MioVisman** [comment]
Domains blocked in "AdGuard Tracking Protection" list
https://github.com/AdguardTeam/AdguardFilters/commit/b380773ebbd8864a9e5b775facedeb5b063a5d2c

**Comment by gwarser** [comment]
If it's region specific then should be blocked by Russian lists.

**Comment by pavelkim** [comment]
I almost peed myself when found these requests on my browser log today. Rostelecom does manipulations with plain http traffic indeed. This is crazy.

**Comment by chestozo** [comment]
wow ... just got the same for the old URL
http://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js

getting the same redirect to
```
http://r.analytic.press/?orig=http%3A%2F%2...
```
